components/scream/src/dynamics/homme/dynamics_driven_grids_manager.cpp

```diff
@@ -198,10 +194,10 @@ build_physics_grid (const std::string& name) {
 
     // Get dimensions and create "empty" grid
     const int nlev  = get_nlev_f90();
-    const int nlcols = get_num_local_columns_f90 ();
-    const int ngcols = get_num_global_columns_f90 ();
+    const int nlcols = get_num_local_columns_f90 (pg_type % 10);
```

Comment:
Does this mean all grids are PG type grids?  Do we plan to only support 10 grid codes, hence the modulo?  If so, why would we allow something to be 11 and map to 1, for example.  Wouldn't we error out?  I'm probably missing something here.